env.sh

```diff
@@ -1,17 +0,0 @@
-#! /bin/sh
```

Comment:
I still use this. Or should we use the development version differently ? 

I guess that part of the code is useless since there is nothing compiled anymore.

It should reduce to  

```bash
#! /bin/sh
ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
export PYTHONPATH="$ROOT:$PYTHONPATH"
```

Reply:
which is maybe not worth a shell script, I consent

Reply:
but isn't adding the commandline tools to the path missing here ?

Reply:
You can simply do `pip install -e .`. The env.sh has been outdated for a while.

Reply:
(Note that this does not work with Meson based installs, which is why here I decided to use flit)

Reply:
The meson-python build backend just added editable installs support. :)

Reply:
Uhh nice, thanks for letting us know. This means editable installs now work for muSpectre and SurfaceTopography...